proposals/0000-no-toplevel-field-selectors.rst

```diff
@@ -234,7 +236,10 @@ None.
 Unresolved questions
 --------------------
 
-- Which order of arguments in the new TH function?
+- What would TH code look like in the future? The disambiguating field name is
+  always the first constructor, maybe introducing a new `Newtype` might make it
```

Comment:
I don't understand the second sentence here. What do you mean by "the disambiguating field name"?

Reply:
Replaced it with "disambiguating record constructor" - clearer?